lisa/nic.py

```diff
@@ -181,11 +181,25 @@ def bind(self, nic: NicInfo, driver_module: str) -> None:
             sudo=True,
         )
 
+    def change_nic_state(self, nic_name: str, state: str) -> None:
```

Comment:
What can stats be changed? If it's a limited set, it's better to have an enumeration.

Reply:
ok sure, it can only be 2 states -  up and down